_layouts/posts.html

```diff
@@ -27,15 +27,29 @@
                     <a href="{{ post.url }}"><p>{{ post.title | escape }}</p></a>
                 {% endif %}
                 </div>
-
-                <p class="subtitle">
+                
+                    <p class="subtitle">
                     <time datetime="{{ post.date | date: date_format }}">{{ post.date | date: date_format }}</time>
                     - {{ authors | remove_first: ', ' | strip }}
-                </p>
+                    </p> 
+                
+        <div class="post-categories">
+            {% if post %}
+              {% assign categories = post.categories %}
+            {% else %}
+              {% assign categories = page.categories %}
+            {% endif %}
+            {% for category in categories %}
+            <a href="{{site.baseurl}}/categories/#{{category|slugize}}" style="color: red; font-style: italic; font-size: 16px; font-family: sans-serif;">{{category | join: ', '}}</a>
```

Comment:
Few comments there:
- I would put a button for each category here instead of a red link. Something like:

   ![image](https://user-images.githubusercontent.com/1842467/196880544-2d021292-fe63-444c-b0bd-eaa874248a04.png)

- Better to use `site.url` here than `site.baseurl`